samples/Mvc.FileUpload/MultipartRequestHelper.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.IO;
+using Microsoft.Net.Http.Headers;
+
+namespace Mvc.FileUpload
+{
+    public static class MultipartRequestHelper
```

Comment:
I will file an issue in HttpAbstractions once I push this PR (want to reference the code that I wrote for this sample to give an idea)